modules/develop/pages/kafka-clients.adoc

```diff
@@ -35,7 +35,7 @@ The following clients have been validated with Redpanda.
 | https://github.com/kafka-rust/kafka-rust[kafka-rust^]
 |===
 
-Clients that have not been validated by Redpanda Data are still compatible, particularly those based on librdkafka.
+Clients that have not been validated by Redpanda Data, but use the Kafka protocol, remain compatible with Redpanda subject to the limitations below (particularly those based on librdkafka, such as confluent-kafka-dotnet or confluent-python).
```

Comment:
> but use the Kafka protocol,

I'm slightly confused by this qualification here because I would assume that all kafka clients use the kafka protocol.

Reply:
@dotnwat Please take a look at https://redpandadata.slack.com/archives/C02FSLH3ZS9/p1705008175705759 and let me know if there is more accurate wording you would like here. thx